compiler/ballerina-lang/src/main/java/org/wso2/ballerinalang/compiler/desugar/Desugar.java

```diff
@@ -2576,8 +2578,8 @@ private void createVarRefAssignmentStmts(BLangRecordVarRef parentRecordVarRef, B
 
     private void createVarRefAssignmentStmts(BLangErrorVarRef parentErrorVarRef, BLangBlockStmt parentBlockStmt,
                                              BVarSymbol errorVarySymbol, BLangIndexBasedAccess parentIndexAccessExpr) {
-        if (parentErrorVarRef.message.getKind() != NodeKind.SIMPLE_VARIABLE_REF ||
-                names.fromIdNode(((BLangSimpleVarRef) parentErrorVarRef.message).variableName) != Names.IGNORE) {
+        if (parentErrorVarRef.message != null && (parentErrorVarRef.message.getKind() != NodeKind.SIMPLE_VARIABLE_REF
```

Comment:
I think we can remove `parentErrorVarRef.message.getKind() != NodeKind.SIMPLE_VARIABLE_REF` as it seems `message` is always a variable reference or null.